packages/client/src/components/Login.js

```diff
@@ -67,6 +97,9 @@ const Login = () => {
             Login
           </Button>
         </ButtonDiv>
+
+        {/* backend errors */}
+        {resErrors && <ErrorDiv> {resErrors} </ErrorDiv>}
```

Comment:
It'd be a better idea to use turnaries, for all of these, I believe this renders the actual word `false` on the screen. The others work because their negative state is `null`, but it's not exactly explicit, it might be a better idea to render using turnaries when using conditionals inside React components in the future. (Instead of shortcutting).
```suggestion
        {resErrors ? <ErrorDiv> {resErrors} </ErrorDiv> : null}
```

Reply:
Ok yes, this makes sense.  We didn't have the issue with "false" displaying on the screen, but I see why the consistency of ternaries is a better practice. Thank you!